src/test/java/com/amazon/opendistroforelasticsearch/security/test/helper/cluster/ClusterHelper.java

```diff
@@ -227,19 +234,28 @@ public void run() {
             throw new RuntimeException("Default template could not be created");
         }
 
+        clusterState = ClusterState.STARTED;
         return cInfo;
     }
 
     public final void stopCluster() throws Exception {
+        closeAllNodes();
+        FileUtils.deleteDirectory(new File("./target/data/"+clustername));
+        clusterState = ClusterState.STOPPED;
+    }
 
+    private final void stopClusterWithoutDeletingData() throws Exception {
+        closeAllNodes();
+        clusterState = ClusterState.STOPPED;
```

Comment:
move to `closeAllNodes()`. It is not necessary to introduce `stopClusterWithoutDeletingData()`, call `closeAllNodes()` from `startCluster()`.